MovieDatabase/MovieDatabase/Data/MovieDbContext.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Microsoft.EntityFrameworkCore;
+using MovieDatabase.Models;
+
+namespace MovieDatabase.Data
+{
+    public class MovieDbContext : DbContext
+    {
+        public MovieDbContext(DbContextOptions<MovieDbContext> options) : base(options) { }
+
+        public DbSet<Movie> Movies { get; set; }
+        public DbSet<Theater> Theaters { get; set; }
+        public DbSet<Worker> Workers { get; set; }
```

Comment:
You need to add the DB configuration for the relationships here.

You can learn about configuring a relationship from this link here: https://www.learnentityframeworkcore.com/configuration/one-to-many-relationship-configuration